### PR TITLE
bug/ICL-335 skipped docs and perf release notes

### PR DESCRIPTION
**Pull request by JGiter**

### CODE REVIEW

File: packages/semantic-release-config/index.js [hunk 1]
```diff
@@ -1,43 +1,55 @@
...
+          noteKeywords: ['BREAKING CHANGE', 'BREAKING CHANGES', 'BREAKING'],
+        },
+        parserConfig: {
+          types: [
+            { type: 'feat', section: 'Features' },
+            { type: 'fix', section: 'Bug Fixes' },
+            { type: 'chore', hidden: true },
+            { type: 'docs', section: 'Documentation' },
+            { type: 'style', hidden: true },
+            { type: 'refactor', hidden: true },
```
Comment (jrhender) [inline packages/semantic-release-config/index.js]: If we're releasing on `style` and `refactor` commits (as configured in the `releaseRules` section above), shouldn't the commits be in the release notes are well?

Comment (JGiter) [reply packages/semantic-release-config/index.js]: yes, probably they should

File: packages/semantic-release-config/index.js [hunk 2]
```diff
@@ -1,43 +1,55 @@
...
-          { type: "docs", release: "patch" },
-          { type: "refactor", release: "patch" },
-          { type: "style", release: "patch" },
-          { type: "test", release: "patch" },
-          { type: "ci", release: "patch" },
-          { type: "build", release: "patch" },
+          { type: 'docs', release: 'patch' },
+          { type: 'refactor', release: 'patch' },
+          { type: 'style', release: 'patch' },
+          { type: 'test', release: 'patch' },
```
Comment (jrhender) [inline packages/semantic-release-config/index.js]: I think we should not release on `test`, then we can keep `test` as `hidden` in the `/release-notes-generator`. What do you think of this?

File: packages/semantic-release-config/index.js [hunk 3]
```diff
@@ -1,43 +1,55 @@
...
+          { type: 'ci', release: 'patch' },
+          { type: 'build', release: 'patch' },
```
Comment (jrhender) [inline packages/semantic-release-config/index.js]: Same with `build`. I don't think we can release on `build` and then not have it be in the `release-notes-generator` as couldn't this lead to a release due to a `build` change but it not being included in the release notes?

File: packages/semantic-release-config/index.js (hunk 2 above)
Comment (JGiter) [reply packages/semantic-release-config/index.js]: Agree

File: packages/semantic-release-config/index.js (hunk 3 above)
Comment (JGiter) [reply packages/semantic-release-config/index.js]: Should we also exclude `ci` from release and notes?

**Review by jrhender — COMMENTED** [review]
@JGiter how does this change relate to our "shared configs". Would this change be better to make in the shared semantic-release config? https://github.com/energywebfoundation/shared-configs/blob/master/packages/semantic-release-config/index.js

**Comment by JGiter** [comment]
> Would this change be better to make in the shared semantic-release config? https://github.com/energywebfoundation/shared-configs/blob/master/packages/semantic-release-config/index.js

I don't follow. I made changes in shared-configs/packages/semantic-release-config/index.js as you suggest

**Comment by jrhender** [comment]
I'm sorry 🤦 . I thought that this PR was in the ICL repo

File: common/changes/@energyweb/semantic-release-config/bug-ICL-335_skipped_release_notes_2022-08-02-11-31.json [hunk 4]
```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@energyweb/semantic-release-config",
+      "comment": "Add documentation and performance commits to release notes",
```
Comment (jrhender) [inline common/changes/@energyweb/semantic-release-config/bug-ICL-335_skipped_release_notes_2022-08-02-11-31.json:5]:
```suggestion
      "comment": "Add documentation and performance commits to release notes. Also add perf and remove ci and build from release rules",
```

**Review by jrhender — COMMENTED** [review]
One other comment is that there also seems to be a styling change here from single quotes to double quotes. Do you know why that is? IMO, it ruins the `git blame` a bit for this file. Would it be possible to execute this change without this styling change, in order to limit the changed lines to the actual "functional" changes?f

File: common/changes/@energyweb/semantic-release-config/bug-ICL-335_skipped_release_notes_2022-08-02-11-31.json (hunk 4 above)
Comment (JGiter) [reply common/changes/@energyweb/semantic-release-config/bug-ICL-335_skipped_release_notes_2022-08-02-11-31.json:5]: Double quotes were changed by single quotes according to our prettier config. We can do it in separate PR though

Comment (JGiter) [reply common/changes/@energyweb/semantic-release-config/bug-ICL-335_skipped_release_notes_2022-08-02-11-31.json:5]: Don't know why git shows changes in eslint-config package. This PR actually doesn't change anything there

**Review by jrhender — APPROVED** [review]
Looks great, thanks @JGiter

**Pull request merged**